docs/user-guide/commands/argocd_app_diff.md

```diff
@@ -27,7 +27,9 @@ argocd app diff APPNAME [flags]
       --local-repo-root string      Path to the repository root. Used together with --local allows setting the repository root (default "/")
       --refresh                     Refresh application data when retrieving
       --revision string             Compare live app to a particular revision
+      --revisions stringArray       Show manifests at specific revisions (comma-separated) for the index of sources in source-indexes
       --server-side-generate        Used with --local, this will send your manifests to the server for diffing
+      --source-indexes int64Slice   List of source indexes. Default is empty array. Indexes start at 1. (default [])
```

Comment:
`intArray` type would make it clearer to users  and not leak go implementation details

Reply:
Unfortunately, this is an auto-generated document. I am using `int64Slice` datatype for sourceIndexes as we do not have a function to parse `int[]` input in flagSet.